index.html

```diff
@@ -0,0 +1,289 @@
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      src="./images/medium.svg"
+                      class="icon m"
+                      alt="medium-icon"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="https://github.com/TekyRey">
+                    <img
+                      src="./images/github.svg"
+                      class="icon"
+                      alt="github-icon"
+                    />
+                  </a>
+                </li>
+                
+                <li>
+                  <a href="#">
+                    <img
+                      src="./images/angel.svg"
+                      class="icon"
+                      alt="angellist-icon"
+                    />
+                  </a>
+                </li>
+              </ul>
+            </div>
+            </div> 
+            
+        
+          </Section>
+<section id="portfolio" class="work-section">
+    <div class="card work">
+  <img class= "mobile-img" src="images/tonic-section.svg" alt="tonic image" aria-hidden="true">
+
+  <div class="content">
+      <h2>Tonic</h2>
+      <div class="experience-section">
+          <p class="canopy">Canopy</p>
+          <img src="images/Counter.svg" alt="" aria-hidden="true">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/Counter.svg" alt="" aria-hidden="true">
+          <p class="bullet">2015</p>
+      </div>
+      <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+      </ul>
+      <button class="submit" type="submit">See Project</button>
+  </div>
+</div>
+
+<div class="card work">
+  <img class= "mobile-img" src="images/availability-section.svg" alt="" aria-hidden="true">
+
+  <div class="content">
+      <h2>Multi-Post Stories</h2>
+      <div class="experience-section">
+          <p class="canopy">Canopy</p>
+          <img src="images/Counter.svg" alt="" aria-hidden="true">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/Counter.svg" alt="" aria-hidden="true">
+          <p class="bullet">2015</p>
+      </div>
+      <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="skills">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+      </ul>
+      <button class="submit" type="submit">See Project</button>
+  </div>
+</div>
+
+<div class="card work">
+  <img class= "mobile-img" src="images/canopy.svg" alt="" aria-hidden="true">
+
+  <div class="content">
+      <h2>Tonic</h2>
+      <div class="experience-section">
+          <p class="canopy">Canopy</p>
+          <img src="images/Counter.svg" alt="" aria-hidden="true">
```

Comment:
I think it would be better if you add an `alt` message for your images, this would help a user understand the content of the image in case it was not rendered on the browser